com.github.linthesia.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "com.github.linthesia",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "linthesia",
+    "finish-args": [
+        "--device=all",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+	"--socket=session-bus",
```

Comment:
please determine which name are need instead of granting the whole session bus. See the `--log-session-bus` option for `flatpak run`